src/containers/CodesplainAppBar.jsx

```diff
@@ -131,20 +132,18 @@ export class CodesplainAppBar extends React.Component {
         label="Discard"
         primary
         onTouchTap={this.handleConfirmNavigation}
-      />
+      />,
     ];
 
-    const { userSnippets } = this.props.userState;
+    const { userSnippets } = this.props;
     const rightElement = this.state.isLoggedIn ?
-      <AppMenu
+      (<AppMenu
```

Comment:
Are these `()` an eslint thing? If not, I don't see why we need to add them, as the `<>` delimit the JSX pretty well.